### PR TITLE
Clarifications on the scope (out of the scope section)

### PR DESCRIPTION
**Pull request by espinr**
In order to clarify the scope of the group:
- a new section (mission) with the introductory part. No changes in the current text, just moved.
- concise scope with bullet points
- new "out of scope" section to specify the specifications won't redefine the existing specifications.

### CODE REVIEW

**Comment by xfq** [comment]
* Preview: https://raw.githack.com/espinr/miniapp/charter-outscope/charters/wg-2020.html
* Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fminiapp%2Fcharters%2Fwg-2020.html&doc2=https%3A%2F%2Fraw.githack.com%2Fespinr%2Fminiapp%2Fcharter-outscope%2Fcharters%2Fwg-2020.html

**Comment by xfq** [comment]
FWIW, the Out of Scope section was removed in #76 for another reason.

File: charters/wg-2020.html [hunk 1]
```diff
@@ -126,23 +126,39 @@ <h1 id="title">
...
     <li>Coordination with other W3C efforts, especially security, privacy, accessibility, internationalization and other Webapp APIs including Progressive Web Apps, on the commonality of the Web.
 </ol>
+
+        <h3 id="out-of-scope">
+          Out of Scope
+        </h3>
+        <p>The Working Group will not:</p>
+        <ul>
+          <li>Define new CSS functionality, syntax, processing or rendering models.</li>
+          <li>Define any specification that overlaps the existing normative specifications for browser-based APIs.</li>
```
Comment (xfq) [inline charters/wg-2020.html:157]: Does MiniApp Lifecycle meet this condition, given https://github.com/w3c/miniapp/blob/gh-pages/specs/lifecycle/docs/explainer.md#comparison-with-some-related-work-in-w3c-such-as-service-worker-page-visibility-and-page-lifecycle ?

Comment (espinr) [reply charters/wg-2020.html]: I agree. Perhaps it's confusing.

**Comment by xfq** [comment]
Related discussions: https://www.w3.org/2020/12/10-miniapp-minutes.html#t01

**Review by tantek — APPROVED** [review]

**Review by muratkalenderoglu — APPROVED** [review]

**Review by travisleithead — APPROVED** [review]
Thanks!

**Pull request merged**